src/metadata/metadata_io.c

```diff
@@ -45,6 +45,19 @@ struct metadata_io_read_i_atomic_context {
 	void *priv;
 };
 
+enum ocf_mio_size {
+	ocf_mio_size_1 = 0,
+	ocf_mio_size_2,
+	ocf_mio_size_4,
+	ocf_mio_size_8,
+	ocf_mio_size_16,
+	ocf_mio_size_32,
+	ocf_mio_size_64,
+	ocf_mio_size_max,
+};
```

Comment:
Most of the enumeration values are unused. single const variable ocf_mio_size_max would be sufficient